wormhole-connect/src/config/mainnet.ts

```diff
@@ -459,6 +483,23 @@ export const MAINNET_TOKENS: { [key: string]: TokenConfig } = {
     decimals: 9,
     solDecimals: 8,
     suiDecimals: 9,
+    seiDecimals: 8,
+  },
+  SEI: {
+    key: 'SEI',
+    symbol: 'SEI',
+    nativeNetwork: 'sei',
+    tokenId: {
+      chain: 'sei',
+      address: 'usei',
+    },
+    icon: Icon.SEI,
+    coinGeckoId: 'sei',
+    color: '#FFFFFF',
+    decimals: 9,
```

Comment:
I want to replace these decimals with a map indexed by the `Context` enum and have `getTokenDecimals` be used exclusively to retrieve this information

Reply:
I agree, we should probably wait for either one of those PRs to be merged and then clean this up